deploy/helm/sumologic/README.md

```diff
@@ -479,6 +476,8 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `pvcCleaner.job.podAnnotations`                                                       | Additional annotations for for the pvcCleaner container.                                                                                                                                                                                                                                                   | `{}`                                                                                                                                                                                 |
 | `pvcCleaner.job.schedule`                                                             | Schedule for cronJobs                                                                                                                                                                                                                                                                                      | `*/15 * * * *`                                                                                                                                                                       |
 | `pvcCleaner.job.securityContext`                                                      | The securityContext configuration for the pvcCleaner.                                                                                                                                                                                                                                                      | `{"runAsUser": 1000}`                                                                                                                                                                |
+| `debug.instrumentation.otelcolInstrumentation.print`                                  | Set to `true` to print logs processed by otelcol-instrumentation collector on standard output.                                                                                                                                                                                                             | `false`                                                                                                                                                                              |
+| `debug.instrumentation.otelcolInstrumentation.stopLogsIngestion`                      | Set to `true` to prevent sending otelcol-instrumentation collector logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of log collector logs.                                                                                                                            | `false`                                                                                                                                                                              |
```

Comment:
```suggestion
| `debug.instrumentation.otelcolInstrumentation.stopLogsIngestion`                      | Set to `true` to prevent sending otelcol-instrumentation collector logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of otelcol-insrumentation logs.                                                                                                                            | `false`                                                                                                                                                                              |
```